### DISCUSSION TITLE
BL touch probe grid centering

### DISCUSSION THREAD

**Discussion by jjfs127** [Jyers/Marlin#60] [Q&A] [answered]
I'm using the `BL Touch 3x3 High Speed` template with `BED_SIZE` set to 230 and notice that the 3x3 probing grid is off-center, would increasing the `PROBING_MARGIN` fix that?
Thanks

**Comment by t3chguy** [comment, marked as answer]
that depends on your probe offset. The ender 3 v2 can actually reach beyond 230 in the X. I think mine managed 259 after I shaved a bolt, so I can get away with a very small probing margin because my BLTouch is mounted very close to the left of the nozzle.

**Reply by jjfs127** [reply]
Thanks, on my previous firmware, I did something like this only through trial and error but thought it was hacky...

```
#define NOZZLE_TO_PROBE_OFFSET {  -46, -8, 0 }

#define X_MIN_POS 0
#define Y_MIN_POS 6.5
#define Z_MIN_POS 0
#define X_MAX_POS X_BED_SIZE
#define Y_MAX_POS (Y_BED_SIZE + 6.5)
#define Z_MAX_POS 250

#define X_BED_SIZE 235
#define Y_BED_SIZE 235
```
Maybe my probe offset it off, but I remembered measuring with calipers and using this duct https://www.thingiverse.com/thing:4637453

**Reply by clintiepoo** [reply]
Teaching Tech's BLTouch [youtube](https://youtu.be/neS7lB7fCww?t=1025) centers it like this: 

**Configuration_adv.h**
```
#if PROBE_SELECTED && !IS_KINEMATIC
  #define PROBING_MARGIN_LEFT 45
  #define PROBING_MARGIN_RIGHT 45
  #define PROBING_MARGIN_FRONT 15
  #define PROBING_MARGIN_BACK 15
#endif

```
